neqo-transport/src/cc/cubic.rs

```diff
@@ -152,6 +152,8 @@ impl WindowAdjustment for Cubic {
         let target_cubic = self.w_cubic(time_ca);
 
         let tcp_cnt = self.estimated_tcp_cwnd / CUBIC_ALPHA;
+        #[allow(unknown_lints)]
+        #[allow(clippy::while_float)]
```

Comment:
There were some conversion errors with your draft code. I think we can do
```rust
let tcp_cnt = self.estimated_tcp_cwnd / CUBIC_ALPHA;
let incr = self.tcp_acked_bytes / tcp_cnt;
self.tcp_acked_bytes -= incr * tcp_cnt;
self.estimated_tcp_cwnd += incr * MAX_DATAGRAM_SIZE_F64;
```
but that makes the `cc::tests::cubic::tcp_phase` test fail with
```
thread 'cc::tests::cubic::tcp_phase' panicked at neqo-transport/src/cc/tests/cubic.rs:191:5:
assertion `left == right` failed
  left: 22
 right: 21
 ```
 because the `expected_ack_cubic_increase` is off by one...

Reply:
I thought I did pretty well considering :)

Is the expectation in the test accurate?  It's hard to know, particular when you look at the comments there.

Reply:
Yeah, I can't tell either.

Reply:
I'm going to change the (unmotivated) `+ 2` in the test to `+ 3`, since I believe the `while`-removal code is actually (more) correct now.

Reply:
Came up with a better change that doesn't require changes to the test.